src/viking.js

```diff
@@ -1,11 +1,53 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength){
+    this.health = health;
+    this.strength = strength;
+  }
+  attack(){
+    return this.strength;
+  }
+  receiveDamage(damage){
+    return this.health -= damage
+  }  
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+  constructor(name, health, strength){
+    super(health, strength)
+    this.name = name
+  }
+  attack(){
+    return this.strength;
+  }
+  receiveDamage(damage){
+    this.health -= damage;
+    if(this.health > 0){
+      return `${this.name} has recived ${damage} points of damage` 
+    } else {
+      `${this.name} has died in act of combat"`
+    }
+  }
+  battleCry(){
+    return "Odin Owns You All!"
+  }
+}
 
 // Saxon
-class Saxon {}
+class Saxon extends Soldier {
+  attack(){
+    super.attack();
+    return this.strength
+  }
+  receiveDamage(damage){
+    this.health -= damage;
+    if(this.health > 0){
+      return `A Saxon has recived ${damage} points of damage`
+    } else {
+      return "A Saxon has died in combat"
+    }
+}
```

Comment:
Your Saxon class is missing closing curly bracket, that's why tests are crashing ;)